src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/AddAzureRmVmssNetworkInterfaceConfigurationCommand.cs

```diff
@@ -106,6 +111,8 @@ private void Run()
                 this.VirtualMachineScaleSet.VirtualMachineProfile.NetworkProfile = new VirtualMachineScaleSetNetworkProfile();
             }
 
+            this.VirtualMachineScaleSet.VirtualMachineProfile.NetworkProfile.NetworkApiVersion = this.NetworkApiVersion?? Microsoft.Azure.Management.Compute.Models.NetworkApiVersion.TwoZeroTwoZeroHyphenMinusOneOneHyphenMinusZeroOne;
```

Comment:
If there is a networkApiVersion already on the NetworkProfile and this.NetworkApiVersion == null, we should preserve the existing networkApiVersion.

Overriding it with the hardcoded version seems more user hostile than helpful.